lib/mapper/variant_mapper.rb

```diff
@@ -9,7 +9,7 @@ def self.map(variant, option_values, product_id, variant_id=nil, not_purchasable
                     price: variant['offers'] && variant['offers'][0] ? variant['offers'][0]['price'] : '0',
                     cost_price: nil,
                     sale_price: nil,
-                    retail_price: nil,
+                    retail_price: variant['offers'] && variant['offers'][0] ? variant['offers'][0]['price'] : '0',
```

Comment:
I think we probably want to leave it as `nil` if it's not provided.

Reply:
@mrogers-5s So like this?